packages/components/src/Banner/Banner.css

```diff
@@ -2,18 +2,13 @@
   display: flex;
   position: relative;
   box-shadow: var(--shadow-base);
-  padding: var(--space-base);
-  color: var(--color-text);
-  background-color: var(--color-surface--background);
-  gap: var(--space-small);
-  align-items: center;
-}
-
-:global(.jobber-retheme) .banner {
   padding: calc(var(--space-base) - var(--space-smaller)) var(--space-base);
   border: var(--border-base) solid var(--color-border);
   border-radius: var(--radius-base);
+  color: var(--color-text);
   background-color: var(--color-surface);
+  gap: var(--space-small);
+  align-items: center;
```

Comment:
That is a great catch, Banner has a lot of React-side styling happening. Will get this cleaned up!

Reply:
Fixed [here](https://github.com/GetJobber/atlantis/pull/1840/commits/5b9eb492051090f58a004afbe80d380a7f5012dd)

Reply:
We don't have to act on it because this conversation is out of scope for this PR, but I'm just noticing that the icon in the web banner is center aligned when there's multiple lines of text, and in the mobile banner the icon is at the top.  

![Screenshot 2024-04-10 at 4 32 25 PM](https://github.com/GetJobber/atlantis/assets/104797704/6ac19a2a-7da4-42ba-83f7-397dc85482e0)
![Screenshot 2024-04-10 at 4 32 35 PM](https://github.com/GetJobber/atlantis/assets/104797704/1c1b8a04-21df-432c-b87c-985b43111847)